app/components/failed_network_action/cloud.js

```diff
@@ -16,7 +16,10 @@ export default class CloudSvg extends PureComponent {
     render() {
         const {color, height, width} = this.props;
         return (
-            <View style={{height, width, alignItems: 'flex-start'}}>
+            <View
+                style={{height, width, alignItems: 'flex-start'}}
+                testID='cloud_icon'
```

Comment:
```suggestion
                testID='failed_network_action.cloud_icon'
```

Reply:
do we need the testID if you are not writing an e2e test?

Reply:
Its not strictly necessary, but it makes writing unit tests easy too. 

It was recommended in a different PR: https://github.com/mattermost/mattermost-mobile/pull/4879#discussion_r502900402